docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
+
+This guide is a walk through to become an [attestation provider](#attestation-provider).
+There are two situations in which it makes sense to do so:
+
+* Running local attestation providers, without any capital requirement, to act as a safeguard against malicious forks ;
+* Running an incentivized attestation provider to reap rewards, which contributes to the network and helps improve the decentralization of the network and provides attestations for the [state connector](#state-connector).
+
+??? note "Rewards"
+
+    Currently, the process of rewarding public attestation providers is still being discussed.
+
+## Prerequisites
+
+Before proceeding with the [installation](#installation), it is essential to follow every prerequisite.
+
+### System Requirements
+
+The following specifications were observed to be able to run all nodes on a single machine.
+Ideally, a single node should be run on each system in production for better performance and uptime.
+
+* **OS**: Ubuntu 20.04
+* **CPU Cores**: 16
+* **RAM**: 96GB
+* **Disk space**: 3TB with an option or plan in place to expand capacity. SSD is recommended because some chains like xrpl can fall out of sync on regular disks.
+
+As of Q2 2022, this is roughly what you can expect from each node regarding disk usage:
+
+| Volume              | Size  |
+|---------------------|-------|
+| algorand-data       | 1TB   |
+| algorand-indexer-db | 1TB   |
+| bitcoin-data        | 500GB |
+| dogecoin-data       | 60GB  |
+| litecoin-data       | 90GB  |
+| ripple-data         | 230GB |
+
+We are looking into ways to run a non-full history algorand node.
```

Comment:
Oh, I never talk about "we" or "us". I prefer to use passive voice.

Also, this sentence should explain _why_ this is being done.

Reply:
I really don't know how to phrase it, and I don't know if it makes sense to mention it in the end 🤔 So I removed the note for now. WDYT?

See [`65c68c5` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/65c68c5650bd96c0109bd07576f796479880e297)

Reply:
I don't think we can just write that you need about 3TB of disk to become an AP (for now) and not say anything about it... How about this:

> In the interest of decentralization, and to allow the maximum number of users to become Attestation Providers, alternate mechanisms are being researched to reduce such high requirements, like using non-full history Algorand nodes.